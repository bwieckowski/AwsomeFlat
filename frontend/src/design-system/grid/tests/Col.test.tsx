import * as React from "react";
import * as renderer from "react-test-renderer";
import ColWrapper from "design-system/grid/Col";
import 'jest-styled-components';

describe("Col", () => {
    it("renders the heading", () => {
        const result = renderer.create(<ColWrapper sizes={[1, 2, 1, 3, 1]}/>).toJSON();
        expect(result).toMatchSnapshot();
    })
});
