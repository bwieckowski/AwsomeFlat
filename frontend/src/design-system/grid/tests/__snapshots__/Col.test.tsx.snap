// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col renders the heading 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

@media (min-width:320px) {
  .c0 {
    width: 100%;
  }
}

@media (min-width:400px) {
  .c0 {
    width: 200%;
  }
}

@media (min-width:768px) {
  .c0 {
    width: 100%;
  }
}

@media (min-width:992px) {
  .c0 {
    width: 300%;
  }
}

@media (min-width:1200px) {
  .c0 {
    width: 100%;
  }
}

@media (min-width:1900px) {
  .c0 {
    width: NaN%;
  }
}

@media (min-width:768px) {
  .c0 {
    padding: 20px;
  }
}

<div
  className="c0"
  sizes={
    Array [
      1,
      2,
      1,
      3,
      1,
    ]
  }
/>
`;
